app/src/main/java/com/dodo/flashcards/presentation/editProfileScreen/EditProfileModels.kt

```diff
@@ -0,0 +1,12 @@
+package com.dodo.flashcards.presentation.editProfileScreen
+
+import com.dodo.flashcards.architecture.ViewEvent
+import com.dodo.flashcards.architecture.ViewState
+
+sealed interface EditProfileViewEvent : ViewEvent {
+    object ClickedEditUsername : EditProfileViewEvent
+    object ClickedEditPassword : EditProfileViewEvent
+    object ClickedReturn : EditProfileViewEvent
+}
+
+object EditProfileViewState : ViewState
```

Comment:
This is awkward right now but in the future, we can put more simple settings that don't require a text field as part of the view state, like if there was a check box to enable a dark mode